CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.53.0
+------------------------------
+*August 1, 2018*
+
+### Changed
```

Comment:
Looks like these should be under `## Added` title, rather than changed, if they are additions